lib/console.js

```diff
@@ -35,12 +35,12 @@ function Console(stdout, stderr, ignoreErrors = true) {
     return new Console(stdout, stderr, ignoreErrors);
   }
   if (!stdout || typeof stdout.write !== 'function') {
-    throw new errors.TypeError('ERR_CONSOLE_WRITABLE_STREAM', 'stdout');
+    throw new ERR_CONSOLE_WRITABLE_STREAM('stdout');
```

Comment:
The only challenge I really have with this is that it makes it far less obvious what *kind* of error is being thrown here.

Reply:
That is also the charming side: you do not have to care about that as it was already taken care about at a earlier step. And we can verify much better that it actually has the correct type. Now you can see the types in the overview in the errors.js file. And I spotted multiple errors due to this.